docs/intro-to-dune/index.md

```diff
@@ -150,80 +173,93 @@ src/counter
 ├─ Counter.re
 ├─ dune
 ├─ Index.re
-└─ Makefile
+└─ index.html
 ```
 
-## Why Makefile
+## Update root directory `index.html`
+
+Now that we've added `src/counter/index.html`, we don't need the root
+directory's `index.html` to render the Counter app. Instead, it can serve as an
+index page to link to all of our single-page apps. Change `index.html` to this:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Melange for React Developers</title>
+  </head>
+  <body>
+    <h1>Melange for React Developers</h1>
+    <ul>
+      <li>
+        <a href="src/counter/index.html">Counter</a>
+      </li>
+    </ul>
+  </body>
+</html>
+```
 
-You might be wondering why we're using a `Makefile` instead of adding another
-npm script. In OCaml projects, usage of [the Make build automation
-tool](https://en.wikipedia.org/wiki/Make_(software)) is common, and this
-includes Melange-related projects. So let's take a short detour to explain the
-`Makefile` we just added.
+Now run `make serve` to check that everything works as expected.
 
-## Anatomy of Makefile
+## Why use Make?
```

Comment:
(Out of scope) Seeing how much complexity it involves to have both make and npm scripts, I wonder if we could just install the opam switch through some script in npm, e.g. `npm run init-opam` or something like that.

Reply:
I can create a separate PR to migrate from Make to npm scripts after this is merged